src/details/ArborX_Dendrogram.hpp

```diff
@@ -13,6 +13,7 @@
 
 #include <ArborX_DetailsDendrogram.hpp>
 #include <ArborX_DetailsKokkosExtSort.hpp>
+#include <ArborX_DetailsUtils.hpp>
```

Comment:
What are we using from it?

Reply:
I had the impression it was not needed.  Remove or add a comment that says what we use from it.

Reply:
Same, sorry about the noise.